### PR TITLE
fix(web): grey out last-active-admin toggles in system users

### PR DESCRIPTION
**Pull request by Fclem**
The system Users page let the last active admin click "Make member" or "Disable" on their own row and only then discover the backend's last-admin guard rejection in an error toast. The role and status toggles are now greyed out on any active-admin row with no other active admin, mirroring the existing backend guard (`ensureAnotherAdmin`) exactly, with backend behavior unchanged.

## Validation

- **E2E (auth project, one worker):** `pnpm e2e:run --project auth tests/auth/` — exit 0, 15 passed. `users-self-actions.spec.ts` covers: sole-admin own row disabled, member and disabled-admin rows enabled, 409 self-PATCH as the sole admin, re-enablement and 200 self-PATCH after a second active admin. Each auth lifecycle spec uses its own per-file database so the full auth project stays deterministic in either file order.
- **Mobile parity (Pixel 5):** `pnpm e2e:run --project mobile-chrome tests/auth/mobile-users-self-actions.spec.ts` — 1 passed; the same disabled/enabled states at the mobile viewport.
- `make typecheck` — passed. `make lint` — passed (golangci 0 issues, eslint `--max-warnings 0`).
- `go test ./internal/auth/... -run TestAdminUserManagement` — passed; the backend guard is untouched.
- `i18n:ratchet` + `i18n:check` — passed (no new user-facing strings).
- Pre-existing environment test gaps documented during the run (proven unrelated by stashing the change): harness-injected `KANDEV_VERSION` breaks two launcher metadata tests, `lib/http-git-server.test.ts` needs a Docker bridge gateway, `test-scripts` needs `unzip`.
- Three rounds of adversarial review (luna); all findings fixed, the final round returned no findings.

Originating task: "Disable self-actions in system users".

## Possible Improvements

Low risk: the button state mirrors the last loaded user list, so a stale list could show an enabled button whose request the backend rejects — the existing 409 toast covers that path unchanged.

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have manually tested my changes and they work as expected.
- [ ] My changes have tests that cover the new functionality and edge cases.
- [ ] If my change touches UI files (`apps/web/`), I have added or updated Playwright e2e tests in `apps/web/e2e/` and verified them with `make test-e2e`.
- [ ] I checked whether this affects public docs in `docs/public/**` and updated them or noted why no docs change is needed.

<a href="https://cubic.dev/pr/kdlbs/kandev/pull/2690?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

## Screenshots

Sole active admin: Make member and Disable greyed out on the own row

After a second active admin: the own row toggles are enabled again

Mobile (Pixel 5): sole active admin, toggles greyed out

### CODE REVIEW

File: apps/web/e2e/tests/auth/mobile-users-self-actions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      KANDEV_DATABASE_PATH: path.join(backend.tmpDir, "kandev-mobile-self-actions.db"),
+    });
+  });
+
+  test.afterAll(async ({ backend }) => {
+    await backend.restart();
+  });
+
+  test("own-row toggles track the last-admin guard on mobile", async ({ browser, backend }) => {
+    const ctx = await browser.newContext({ baseURL: backend.frontendUrl });
```
[the remark this answers is not in the document]
Comment (Fclem) [reply apps/web/e2e/tests/auth/mobile-users-self-actions.spec.ts]: Fixed in 581a62b7b: the mobile spec now spreads devices["Pixel 5"] into browser.newContext (matching mobile-menu-theme-toggle.spec.ts) and pins the viewport with an assertion (expect(page.viewportSize()?.width).toBe(393)) so the device-options regression cannot silently recur. The width assertion also caught that headless mobile emulation reports 393x727, not 393x851, so only the width is pinned. Verified: pnpm e2e:run --project mobile-chrome tests/auth/mobile-users-self-actions.spec.ts -> 1 passed at the real Pixel 5 width. The same bug had affected the disposable capture spec, so the PR's mobile screenshot was desktop-width; it was recaptured at 393px and re-embedded (media ref updated to 448e0031).
